self_teach_challenge_1.1/script.js

```diff
@@ -12,7 +16,53 @@ function stopCountdown() {
     }
 };
 
+function countdownSettings() {
+    if (disable) {
+        document.getElementById('minutes').disabled = false;
+        document.getElementById('seconds').disabled = false;
+        disable = false;
+    } else {
+        document.getElementById('minutes').disabled = true;
+        document.getElementById('seconds').disabled = true;
+        disable = true;
+    }
+}
+
+function changeButtonValue() {
+    let button = document.getElementById('timer-toggle-button').value;
+    if (clicked) {
+        /*button.value ='STOP' */
+        document.getElementById('timer-toggle-button').value = 'STOP';
+        clicked = false;
+    } else {
+        /*button.value ='START' */
+        document.getElementById('timer-toggle-button').value = 'START';
+        clicked = true;
+    }
+    document.getElementById('minutes').disabled = true;
+    document.getElementById('seconds').disabled = true;
+}
+
+function oneNumber() {
```

Comment:
Consider: function names should contain a verb, so that it states its intention.